librarian_resources/BookRecognition recognition

int32 database_id

float32 confidence

std_msgs/String title